.travis.yml

```diff
@@ -109,12 +110,8 @@ script:
   - py.test
   - python setup.py bdist_wheel
   - pip install ./dist/coala-*.whl
-  - pip install coala-bears[alldeps] --pre -U
-  # https://github.com/coala/coala-bears/issues/1037
-  - >
-    if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then
-      sed -i.bak '/bears = GitCommitBear/d' .coafile
-    fi
+  - curl -fsSL -o coala-bears.zip $BEARS_ZIP_URL/master
+  - pip install coala-bears.zip[alldeps] --pre -U
```

Comment:
Btw, I still quite don't understand about this. Doesn't `pip install coala-bears[alldeps] --pre -U` already install the master branch of `coala-bears` ?

Reply:
No. It installs the pre-release in pypi, from January 2018.

We still are not ready to do pre-releases yet, with coala-utils and dependency-management still needing more work and need to be released first.